src/reducer/cartReducer.jsx

```diff
@@ -0,0 +1,35 @@
+export const addToCartReducer=(state, dispatch)=>{
```

Comment:
Just a suggestion, generally we use `action` keyword instead of `dispatch` in the reducer function.